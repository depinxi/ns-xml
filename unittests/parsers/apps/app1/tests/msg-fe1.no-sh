CLI: "-av", "v is an invalid option"
Value count: 1
Values: "v is an invalid option"
Error count: 1
Subcommand: 
switchA=True
argumentB=
