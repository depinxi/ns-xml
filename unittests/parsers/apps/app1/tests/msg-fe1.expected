CLI: "-av", "v is an invalid option"
Value count: 0
Values: 
Error count: 1
Subcommand: 
switchA=True
argumentB=
